### DISCUSSION TITLE
Does 'RandomFlip3D' flip images?

### DISCUSSION THREAD

**Discussion by anhquan3012** [open-mmlab/mmdetection3d#2309] [General] [answered]
I am trying to implement a multimodal detector on mmdet3d. Does setting 'RandomFlip3D' in the pipeline flips the images also? Or it just flips the point cloud? Also, does the flip works with multi-view images (i.e. will left image become right image and vice versa)?

**Comment by Tai-Wang** [comment, marked as answer]
RandomFlip3D would flip the image following the typical way in 2D but not swap the left and right images. The flip of the 2D image is conducted by inheriting the 2D transform RandomFlip.
